app/post_helpers_test.go

```diff
@@ -0,0 +1,295 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package app
+
+import (
+	"testing"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_getTimeSortedPostAccessibleBounds(t *testing.T) {
+	var p = func(at int64) *model.Post {
```

Comment:
`p` is a bad choice for a function name. A better name would be postFromCreateAt. Remember that - "further the variable is invoked from the declaration site, the longer it should be": https://github.com/golang/go/wiki/CodeReviewComments#variable-names